osbenchmark/benchmark.py

```diff
@@ -299,14 +299,14 @@ def add_workload_source(subparser):
              "Check https://opensearch.org/docs/version-history/ for released versions.",
         default="")
     install_parser.add_argument(
-        "--provision-config-instance",
-        help=f"Define the provision_config_instance to use. List possible "
-        f"provision_config_instances with `{PROGRAM_NAME} list "
-        f"provision_config_instances` (default: defaults).",
+        "--cluster-config",
+        help=f"Define the cluster_config to use. List possible "
```

Comment:
Nit: `cluster_config` and `cluster_configs` --> `cluster-config` and `cluster-config`